app/theme/client/imports/components/messages.css

```diff
@@ -43,6 +43,11 @@
 		}
 	}
 }
+.rc-ui-kit {
+	display: inline-block;
+	width: 100%;
+    max-width: 400px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/app/RocketChat/Rocket-Chat/pullRequest?prid=4651020)